src/components/Table/TableCell/TableCell.module.scss

```diff
@@ -1,4 +1,6 @@
 .tableCell {
   padding: 9px 16px;
   overflow: hidden;
+  display: flex;
+  align-items: center;
```

Comment:
if already aligning the content, it should be ok to remove the padding

Reply:
done

Reply:
@shlomitc the content is only aligned vertically. I would recommend keeping the padding, otherwise the design would look like this:

<img width="965" alt="Screenshot 2023-11-23 at 09 28 25" src="https://github.com/mondaycom/monday-ui-react-core/assets/144025856/82079cc6-3b1c-4902-be62-5ca95e229e9f">

Reply:
so let's keep the left padding and remove the rest

Reply:
fixed!